plugins/labhub.py

```diff
@@ -214,6 +214,8 @@ def mark_cmd(self, msg, match):
             return 'Repository doesn\'t exist.'
         else:
             current_labels = list(mr.labels)
+            data = mr._get_data()
```

Comment:
Please don't use private functions. Since, it is not mandatory that you'll get a response of same schema from all providers like GitHub, GitLab, etc.